bagger/config/default.example.toml

```diff
@@ -0,0 +1,23 @@
+[Defaults]
+output_dir = "out"
+workflow = "redata_preservation/config/default_workflow.json"
```

Comment:
update the path to match the new path? Also, is this path relative to the main ReBACH directory or to the main ReBACH-bagger script?

Reply:
Fixed. The readme mentions that all paths in the config are relative to the directory from which bagger is invoked. For instances where the bagger module is imported into other scripts, this can get ambiguous, so I suggest using absolute paths if necessary. I can include comment to this effect in the example config file itself if that's helpful.